index.html

```diff
@@ -13524,16 +13524,18 @@ <h3>Presentational Roles Conflict Resolution</h3>
 		<ul>
 			<li>If an element is focusable, user agents MUST ignore the <rref>none</rref>/<rref>presentation</rref> role and expose the element with its implicit role, in order to ensure that the element is <a>operable</a>.</li>
 			<li>If an <a href="#mustContain">allowed child element</a> has an explicit non-presentational role, user agents MUST ignore an inherited presentational role and expose the element with its explicit role. If the action of exposing the explicit role causes the accessibility tree to be malformed, the expected results are undefined.</li>
-      <li>If an element has <a>global</a> WAI-ARIA states or properties, user agents MUST ignore the <rref>none</rref>/<rref>presentation</rref> role and instead expose the element's implicit role. However, if an element has only non-global, role-specific WAI-ARIA states or properties, the element MUST NOT be exposed unless the presentational role is inherited and an explicit non-presentational role is applied.</li>
+			<li>
+				<p>If an element with an explicit presentational role has <a>global</a> WAI-ARIA states or properties, user agents MUST ignore the <rref>none</rref>/<rref>presentation</rref> role and instead expose the element's implicit role. Conversely, if the presentational role is inherited and the element has <a>global</a> WAI-ARIA states or properties, user agents MUST NOT expose the element, maintaining its presentational role regardless of any <a>global</a> ARIA state or property. However, if an element has only non-global, role-specific WAI-ARIA states or properties, the element MUST NOT be exposed unless the presentational role is inherited and an explicit non-presentational role is applied.</p>
```

Comment:
>the element MUST NOT be exposed unless the

the element or the element's role?

Reply:
Good point, Scott!
While looking through the ARIA specifications, I found similar wording and re-used it, assuming the intent was to not expose the element's role rather than the element itself (I can't recall the exact location right now). This interpretation seems supported by the following statement:

> Thus, the [none](https://w3c.github.io/aria/#none)/[presentation](https://w3c.github.io/aria/#presentation) role causes a given element to be treated as having no role or to be removed from the [accessibility tree](https://w3c.github.io/aria/#dfn-accessibility-tree), but does not cause the content contained within the element to be removed from the accessibility tree.

But I'm more than happy to use your wording to clarify that we do not expect the content to be ignored, only the element's role.

Reply:
Oh, I found it! It was the original statement I didn't change:

> However, if an element has only non-global, role-specific WAI-ARIA states or properties, the element MUST NOT be exposed unless the presentational role is inherited and an explicit non-presentational role is applied.

I'll update both